src/libraries/System.Linq.Queryable/src/System/Linq/EnumerableExecutor.cs

```diff
@@ -29,13 +29,13 @@ public EnumerableExecutor(Expression expression)
             _expression = expression;
         }
 
-        internal override object ExecuteBoxed() => Execute();
+        internal override object? ExecuteBoxed() => Execute();
 
         internal T Execute()
         {
             EnumerableRewriter rewriter = new EnumerableRewriter();
             Expression body = rewriter.Visit(_expression);
-            Expression<Func<T>> f = Expression.Lambda<Func<T>>(body, (IEnumerable<ParameterExpression>)null);
+            Expression<Func<T>> f = Expression.Lambda<Func<T>>(body, (IEnumerable<ParameterExpression>?)null);
```

Comment:
System.Linq.Expressions hasn't been annotated yet.  There's an open PR for it.  I fully expect that once that goes in, this will need to be revisited.  Please do not merge this PR until after that's in (in general we should avoid working on libraries that depend on others that haven't been annotated... this looks like a mistake I contributed to, in that my tool which analyzed the dependencies in https://github.com/dotnet/corefx/issues/40623 apparently missed this one... sorry about that).